release-team/role-handbooks/release-notes/README.md

```diff
@@ -179,8 +176,8 @@ To update an entry in this section the following steps must be performed:
 ### Week 1
 
 Begin running release-notes tool for the ongoing collection of release notes with the first alpha release, which has been cut directly after the latest minor.
-    - Update the `release-notes-draft.md`
-    - Update the website
```

Comment:
Do we want to verify at least that the notes are available on the website?

Reply:
That's a good idea. Currently, `krel release` publishes artificates to GCS which should instantaneously be picked up by the website. So is this the correct spot to add that?

Reply:
Yes, I think so :+1: